### DISCUSSION TITLE
Vaccination data not updated? (April 2 9:19am EST)

### DISCUSSION THREAD

**Discussion by philipschellekens** [owid/covid-19-data#2581] [General]
Seems like the COVID-19 vax dataset is not updated today.

**Comment by edomt** [comment]
Hi @fibke 

We’re considering switching to updating our vaccination dataset only on weekdays, and so this weekend is a trial run of this change. The vaccination data will be next updated on Monday morning (with 3 new days of data).

**Reply by philipschellekens** [reply]
Thanks @edomt. Definitely would be in favor of maintaining the daily publication schedule given OWID's status as main data provider and the fact that we still have a long way to go on vaccination.
